status_bar_view.py

```diff
@@ -70,15 +70,24 @@ def on_selection_modified_async(self, view):
 
 
 def draw(active_view, we_count, current_pos, errors_per_line, **kwargs):
-    if not we_count:
+    counters = persist.settings.get('statusbar.counters')
+    messages = persist.settings.get('statusbar.messages')
+    msgs = messages_under_cursor(errors_per_line, current_pos)
```

Comment:
These two variable names (messages, msgs) are confusing. If "show_" is added to the settings, it should be for the variables as well.